src/components/ScoreDelta/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useMemo } from 'react';
+import styled from 'styled-components';
+import { partialRight, pipe, prop } from 'ramda';
+
+import { SSCIconNames } from '../../theme/icons/icons.enums';
+import { Icon } from '../Icon';
+import { ScoreDeltaProps } from './ScoreDelta.types';
+import { Inline } from '../layout';
+import { getColor, getFontSize } from '../../utils';
+import { ColorTypes } from '../../theme';
+
+const IconRotation = {
+  positive: '45deg',
+  negative: '135deg',
+  neutral: '0deg',
+};
+
+const IconBackgroundColor = {
+  positive: ColorTypes.success100,
+  negative: ColorTypes.error100,
+  neutral: ColorTypes.neutral300,
+};
+
+const FontColor = {
+  positive: ColorTypes.success500,
+  negative: ColorTypes.error500,
+  neutral: ColorTypes.neutral600,
+};
+
+const IconWrapper = styled.div<{ type: string }>`
+  transform: rotate(${pipe(prop('type'), partialRight(prop, [IconRotation]))});
+  background: ${pipe(
+    prop('type'),
+    partialRight(prop, [IconBackgroundColor]),
+    getColor,
+  )};
+  width: 20px;
+  height: 20px;
+  border-radius: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const Label = styled.span<{ type: string }>`
+  font-size: ${getFontSize('md')};
+  color: ${pipe(prop('type'), partialRight(prop, [FontColor]), getColor)};
+`;
+
+const getType = (delta: number) => {
+  if (delta > 0) return 'positive';
+  if (delta < 0) return 'negative';
+  return 'neutral';
+};
+
+const getDecimals = (number, count) => {
+  const rest = number % 1;
+  return Math.floor(rest * 10 ** count)
+    .toString()
+    .padEnd(count, '0');
+};
+
+const ScoreDelta = React.forwardRef<HTMLDivElement, ScoreDeltaProps>(
+  ({ delta, decimalsCount = 0 }, ref) => {
+    const icon = delta === 0 ? SSCIconNames.minus : SSCIconNames.arrowUp;
+    const type = getType(delta);
+    const coeficient = Math.floor(Math.abs(delta));
+    const decimals = getDecimals(delta, decimalsCount);
+    const text = useMemo(() => {
+      return [coeficient, decimalsCount > 0 ? decimals : undefined]
+        .filter((item) => item !== undefined)
+        .join('.');
+    }, [coeficient, decimals, decimalsCount]);
```

Comment:
This seems unnecessarily complex. You could just use:
`const label = Math.abs(Number.parseFloat(delta).toFixed(decimalsCount))`